<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookSeat</class>
 <widget class="QMainWindow" name="BookSeat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1911</width>
    <height>1006</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>预定座位</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/res/0003.png</normaloff>:/res/0003.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget{
	background-image: url(:/res/LW (2).jpg);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>1501</width>
      <height>831</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
*{
    background:rgba(255,255,157,0.80);
	font-size:20px;
	color:black;
	font-family:Microsoft JhengHei;
}
QHeaderView::section{
    background:rgba(255,255,157,0.90);
	font-size:20px;
	color:black;
	font-family:Microsoft JhengHei;
}
QHeaderView{
	min-height:30px;
    background:rgba(255,255,157,0.80);
	font-size:20px;
	color:black;
	font-family:Microsoft JhengHei;
}

</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>30</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{
font-size:40px;
font-family:Microsoft JhengHei;
background:rgba(255,255,157,0.50);
border-radius:0px;
}



/* 下拉后，整个下拉窗体越过每项的样式 */
QComboBox QAbstractItemView::item:hover {
color:black;
font-size:40px;
font-family:Microsoft JhengHei;
background:rgba(0,0,0,0);
border-radius:0px;  /* 整个下拉窗体越过每项的背景色 */
}


QComboBox QAbstractItemView::item:selected {
font-size:40px;
font-family:Microsoft JhengHei;
background:rgba(255,255,157,0.50);
border-radius:0px;
}</string>
    </property>
    <item>
     <property name="text">
      <string>楼名</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>楼号</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>层数</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>教室号</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>教室id</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>30</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background:rgba(255,255,157,0.70);
border-radius:26px;
}
QPushButton:hover{
background:#FFE11A;
border-radius:26px;

}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/res/0003.png</normaloff>:/res/0003.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>481</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
font-size:40px;
font-family:Microsoft JhengHei;

background:rgba(255,255,157,0.50);
border-radius:25px;
}



QLineEdit::hover
{

  color:green
  border-color:rgb(50,480,40);
  background-color:green;
}
</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>   搜索</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>1580</x>
      <y>590</y>
      <width>291</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background:rgba(255,225,26,0.5);
border-radius:15px;
color:#000000;
font-size:40px;
font-family:Microsoft JhengHei;
border:2px solid #002D40;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>输入教室id</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>1590</x>
      <y>690</y>
      <width>121</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:#96CA2D;
border-radius:60px;
font-size:28px;
font-family:Microsoft JhengHei;
}
QPushButton:hover{
background:#168039;
border-radius:60px;
font-size:28px;
font-family:Microsoft JhengHei;
}
</string>
    </property>
    <property name="text">
     <string>预定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>1740</x>
      <y>690</y>
      <width>121</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:#ff8c69;
border-radius:60px;
font-size:25px;
font-family:Microsoft JhengHei;
}
QPushButton:hover{
border-radius:60px;
background:#FF6F69;
font-size:25px;
font-family:Microsoft JhengHei;
}
</string>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>30</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background:rgba(255,255,157,0.70);
border-radius:15px;
font-size:30px;
font-family:Microsoft JhengHei;
}
QPushButton:hover{
background:#FFE11A;
border-radius:15px;
font-size:30px;
font-family:Microsoft JhengHei;
}
</string>
    </property>
    <property name="text">
     <string>显示全部</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dateTimeEdit">
    <property name="geometry">
     <rect>
      <x>1670</x>
      <y>470</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2020</year>
      <month>6</month>
      <day>1</day>
     </datetime>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dateTimeEdit_2">
    <property name="geometry">
     <rect>
      <x>1670</x>
      <y>530</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2020</year>
      <month>7</month>
      <day>1</day>
     </datetime>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1580</x>
      <y>470</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>起始时间：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1580</x>
      <y>530</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>结束时间：</string>
    </property>
   </widget>
   <zorder>lineEdit</zorder>
   <zorder>tableView</zorder>
   <zorder>comboBox</zorder>
   <zorder>pushButton</zorder>
   <zorder>lineEdit_2</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>pushButton_5</zorder>
   <zorder>dateTimeEdit</zorder>
   <zorder>dateTimeEdit_2</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
